### ISSUE TITLE
Migration to Oracle DB executes but fails with RuntimeSqlException and java.sql.SQLWarning: Warning: execution completed with warning

### ISSUE THREAD

**Issue by svscorp** [mybatis/migrations#91]
I am using mybatis-migrations 3.3.1 and Oracle DB.

Besides changelog.sql I have one migration file with a one-liner. If I run `migrate up` the log shows 
"failure" while SQL is actually being executed.

20171013000000_test.sql
````
-- // test 

CREATE TABLE test AS (SELECT * FROM parameters);

-- //@UNDO

DROP TABLE test;
````

migrate up
````
~/mybatis/mybatis-migrations-3.3.1> migrate up
------------------------------------------------------------------------
-- MyBatis Migrations - up
------------------------------------------------------------------------
========== Applying: 20170101000000_create_changelog.sql =======================
--  Create Changelog
CREATE TABLE CHANGELOG (
ID NUMERIC(20,0) NOT NULL,
APPLIED_AT VARCHAR(25) NOT NULL,
DESCRIPTION VARCHAR(255) NOT NULL
)
ALTER TABLE CHANGELOG
ADD CONSTRAINT PK_CHANGELOG
PRIMARY KEY (id)
========== Applying: 20171013000000_test.sql ======================
--  test
CREATE TABLE test AS (SELECT * FROM parameters)
Error executing: CREATE TABLE test AS (SELECT * FROM parameters)
.  Cause: java.sql.SQLWarning: Warning: execution completed with warning
------------------------------------------------------------------------
-- MyBatis Migrations FAILURE
-- Total time: 12s
-- Finished at: Mon Oct 16 12:53:01 CEST 2017
-- Final Memory: 18M/483M
------------------------------------------------------------------------
ERROR: Error executing command.  Cause: org.apache.ibatis.jdbc.RuntimeSqlException: Error executing: CREATE TABLE test AS (SELECT * FROM parameters)
.  Cause: java.sql.SQLWarning: Warning: execution completed with warning
````

Why is that?

**Comment by harawata** [comment]
Hi @svscorp ,

Please see my comments on #84 .
That issue is about PostgreSQL, but the reason and the solution are the same.

It seems that DB servers raise warnings more casually than I thought. 😵
Maybe we should consider changing the default value of `ignore_warnings` from `false` to `true`.
@h3adache What's your take on this?

**Comment by h3adache** [comment]
I think changing the default to true makes a lot of sense.
A warning shouldn't be treated as an error unless explicitly told to do so.
I looked at the ticket that this behavior was added for and it looks like it was caused by a bad script that Oracle couldn't compile but only warned about.
I think having to set it false to handle that special case would be fine.

**Comment by svscorp** [comment]
Hi @h3adache @harawata 

Nice. I'll check on that and close the issue if it is fixed. Agreeing on suggested default value change.

Thank you.

**Comment by jpenalver10** [comment]
Hi @harawata @h3adache 

I agree "A warning shouldn't be treated as an error unless explicitly told to do so.".

Set default value to true its meaning that I can not set the propertie? or the default value in environment.properties template?

thanks

**Comment by harawata** [comment]
@h3adache 
Totally makes sense!
I'll make the change and add a comment on related issues.
Thank you very much!

@svscorp @jpenalver10
Thank you guys for the comments!

> Set default value to true its meaning that I can not set the propertie? or the default value in environment.properties template?

The default value in the environment.properties template is changed to `true`.
And even if there is no line in the properties file, it's treated as `true`.

**Comment by svscorp** [comment]
Hi @harawata @h3adache 

Me and the team has validated - it works.

Thank you! Will you release 3.3.2 soon ? :)

**Comment by svscorp** [comment]
Ping @harawata @h3adache

**Comment by harawata** [comment]
@svscorp ,

Sorry for a late reply.
To apply the fix for #96 , it may have to wait for MyBatis 3.4.6 to be released.

There is no full time committer in the team, so please bear with us for the delay!

**Comment by svscorp** [comment]
@harawata no problems, I totally understand it. I was patiently waiting :)

Thank you for support!

**Comment by jpenalver10** [comment]
Hi @harawata !

do you know when you releases the new version that resolve this issue?

thanks!

**Comment by svscorp** [comment]
Hi @jpenalver10 I think it is available in latest snapshot (3.3.2-SNAPSHOT).

**Comment by harawata** [comment]
Hi @jpenalver10 ,

Migrations has to wait for the next core release (3.4.6), but there still is an ongoing work there.
And @svscorp is right. It would be really helpful if you could try the SNAPSHOT to make sure everything works as you expect.
